symengine/lambda_double.h

```diff
@@ -451,6 +451,49 @@ class LambdaRealDoubleVisitor
             return result;
         };
     };
+
+    void bvisit(const Infty &x)
+    {
+        if (x.is_negative_infinity()) {
+            result_ = [=](const double * /* x */) {
+                return -std::numeric_limits<double>::infinity();
+            };
+        } else if (x.is_positive_infinity()) {
+            result_ = [=](const double * /* x */) {
+                return std::numeric_limits<double>::infinity();
+            };
+        } else {
+            throw SymEngineException(
+                "LambdaDouble can only represent real valued infinity");
+        }
+    }
+
+    void bvisit(const Contains &x)
+    {
+        x.get_expr()->accept(*this);
+        x.get_set()->accept(*this);
```

Comment:
Looks like you need to implement this for each type of set we have including Interval.